libqtile/widget/check_updates.py

```diff
@@ -39,6 +39,8 @@
     "Fedora": ("dnf list updates -q", 1),
     "FreeBSD": ("pkg_version -I -l '<'", 0),
     "Mandriva": ("urpmq --auto-select", 0),
+    "void": ("xbps-install -nuMS", 0),
+    "VoidLinux": ("xbps-install -nuMS", 0),
```

Comment:
Why do we need 2 entries here? The command seems to be the same.

Reply:
It was just because the people could get the values from different piping commands. But I will leave just one.

Reply:
Can we just call it "Void"?

Reply:
I think so. I could set an environment variable in case I change distributions in the future.